site/content/blogs/release-v127.en.md

```diff
@@ -87,11 +87,74 @@ Continue with the deployment? (y/N)
 
 If the diff looks good to you, enter "y" to deploy. Otherwise, enter "N" to make adjustments as needed!
 
-## Enable multiple listeners and routing rules for Load Balancers
 
-### Add multiple host-based or path-based routing rules to your Application Load Balancers
+## Enable multiple listeners and listener rules for Load Balancers
+You can now expose multiple ports through your Load Balancers.
```

Comment:
nit: this description gives me the impression that "an ALB can listen to multiple ports, not just 80/443", which is not possible. 

I wonder what description would be more interesting to users (what are their end goals) ? Would they be interested in:
1. Exposing multiple container port, or
2. Being able to handle traffics from different path, host, or port using one service

Maybe it's the latter 🤔 ? 


I will throw out an idea here just to spin up the thinking 💭 :
```suggestion
A Load-Balanced Web Service or a Backend Service can now handle more variety of traffic now.
```

Reply:
I agree that the main value prop isn't "exposing more ports", we should reframe everything to be about exposing additional listener rules for ALBs and listeners for NLBs.